frontend/src/components/TaskDetailsBody/TaskDetailsBody.tsx

```diff
@@ -16,25 +16,27 @@ import { Dropdown } from '../dropdown/CustomDropDown';
 
 type TaskDetailsBodyType = {
   task: Task;
+  getAllTasks: () => void;
 };
 
-const TaskDetailsBody = ({ task }: TaskDetailsBodyType) => {
+const TaskDetailsBody = ({ task, getAllTasks }: TaskDetailsBodyType) => {
   const { completed, setCompleted } = useCompletedStore();
   const { setTask } = useTaskStore();
   const completedStatus = completed[`${task.id}`] ?? false;
   const [title, setTitle] = useState(task.title);
   const [description, setDescription] = useState(task.description);
   const defaultDueDate = task.due_date ? format(new Date(task.due_date), "yyyy-MM-dd'T'HH:mm") : '';
   const [dueDate, setDueDate] = useState(defaultDueDate);
+  const [projectUsers, setProjectUsers] = useState([]);
```

Comment:
<img width="646" alt="image" src="https://github.com/HackYourFuture-CPH/class-25-lothbrok/assets/5738476/469622a0-598b-45b3-9ae4-84a3ad358987">
some issues to look into